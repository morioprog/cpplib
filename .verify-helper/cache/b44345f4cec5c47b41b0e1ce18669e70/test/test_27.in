6
1 1
-1 1
-5 7
-3 2
-4 7
1 5
