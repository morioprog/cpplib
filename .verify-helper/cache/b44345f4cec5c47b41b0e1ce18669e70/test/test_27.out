1 1
1 5
-4 7
-5 7
-1 1
-3 2
